src/sys/unix.rs

```diff
@@ -1618,6 +1618,68 @@ impl crate::Socket {
         }
     }
 
+    /// Get the value of the `SO_RCVLOWAT` option on this socket.
+    ///
+    /// For more information about this option, see [`set_rcv_lowat`].
+    ///
+    /// [`set_rcv_lowat`]: crate::Socket::set_rcv_lowat
+    // TODO: add Redox support, but I'm not sure if Redox supports SO_RCVLOWAT
```

Comment:
If Redox doesn't support it that's too bad, no need to add these TODOs.

Reply:
removed